### PR TITLE
Adds optimize LCP article

### LINKED ISSUE #2704: content: optimize LCP
* Author: @housseindjirdeh 
* Stakeholders: @philipwalton, @addyosmani 
* [Draft](https://docs.google.com/document/d/1-wixEOMo6L1iHievoAUvaUz2isLym94vfNT6UI0WB-k/edit)

### PR DESCRIPTION
**Pull request by housseindjirdeh**
Fixes #2704

Changes proposed in this pull request:

- Adds optimize LCP article ⚡

### CODE REVIEW

**Comment by kaycebasques** [comment]
@housseindjirdeh FYI I created a separate issue (#2704) to track this new page of content (we generally do one issue per new page of content to get a better idea of how much new content is landing)

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+</figure>
+
+{% Aside %}
+For a deeper dive into different server-rendering architectures, take a look at [Rendering on the
+web](https://developers.google.com/web/updates/2019/02/rendering-on-the-web).
+{% endAside %}
+
+## Developer tools
+
+A number of tools are available to measure and debug Largest Contentful Paint (LCP).
```
Comment (kaycebasques) [inline src/site/content/en/blog/optimize-lcp/index.md:627]:
```suggestion
A number of tools are available to measure and debug LCP:
```

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 2]
```diff
@@ -0,0 +1,644 @@
...
+    measuring LCP in a lab setting.
+
+    ![Lighthouse 6.0](./lighthouse.jpg)
+
++   The **Timings** section of the
+    [Performance](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance)
+    panel in Chrome DevTools includes a LCP marker and shows you which element is associated with
+    LCP when you hover over the **Related Node** field.
+
+    ![LCP in Chrome DevTools](./lcp-chrome-devtools.png)
```
Comment (kaycebasques) [inline src/site/content/en/blog/optimize-lcp/index.md:639]: This image is OK because the text that's essentially an ad for a Google product is pretty small and not noticeable.

**Comment by philipwalton** [comment]
@housseindjirdeh  I noticed that the threshold image in your post uses the older "adequate" label. This should be updated to "Needs Improvement". (Not sure who owns this graphic, but I'm going to be creating similar graphics, so let me know if I can help here.)

cc: @addyosmani to ensure his article does the same.

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 3]
```diff
@@ -0,0 +1,644 @@
...
+and only go to the network when the file's content has changed.
+
+The following chart shows how LCP distributions have been reduced on a site using this pattern: 
+
+<figure class="w-figure">
+  <img src="./lcp-sw-caching.png" alt="Largest Contentful Paint distributions before and after HTML caching">
+  <figcaption class="w-figcaption">Largest Contentful Paint distribution, with and without a service worker - <a href="https://philipwalton.com/articles/smaller-html-payloads-with-service-workers/">philipwalton.com</a></figcaption>
+</figure>
+
+This chart shows the number of page loads on a single site within the past 30 days with varying LCP
```
Comment (philipwalton) [inline src/site/content/en/blog/optimize-lcp/index.md:117]: Suggest rewording: "The chart shows the distribution for LCP from a single site over the last 28 days, segmented by service worker state. Notice how far more page loads have a faster LCP value after cache-first HTML page serving was introduced in the service worker (blue portion of chart)."

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 4]
```diff
@@ -0,0 +1,644 @@
...
+values. Notice how far more page loads have a faster LCP value after HTML caching was introduced
+using a service worker (blue portion of chart).
+
+{% Aside %}
+To learn more about how service workers can serve HTML content cache-first, take a look at [Smaller
```
Comment (philipwalton) [inline src/site/content/en/blog/optimize-lcp/index.md:122]: Suggest rewording: "To learn more about techniques for serving full or partial HTML pages cache-first, take a look at..."

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 5]
```diff
@@ -0,0 +1,644 @@
...
+  Largest Contentful Paint (LCP) can be used to determine when the main content of the page has
+  finished rendering on the screen. Learn how to optimize LCP by improving slow server response
+  times, resource load times and client-side rendering.
+tags:
+  - post
+  - performance
+  - web vitals
+---
+
+"I can't see any useful content! Why does it take so long to load? 😖"
```
[the remark this answers is not in the document]
Comment (housseindjirdeh) [reply src/site/content/en/blog/optimize-lcp/index.md]: Nice, that looks better :)

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 6]
```diff
@@ -0,0 +1,644 @@
...
+rendering on the screen.
+
+<img class="w-screenshot w-screenshot--filled" src="lcp-distributions.png" alt="Good, adequate, and poor LCP values">
+
+The most common causes of a poor LCP are:
+
++   Slow server response times
++   Render-blocking JavaScript and CSS
++   Resource load times
++   Client-side rendering
```
[the remark this answers is not in the document]
Comment (housseindjirdeh) [reply src/site/content/en/blog/optimize-lcp/index.md]: Good thinking, updated!

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 7]
```diff
@@ -0,0 +1,644 @@
...
+
+{% Aside %}
+Check out [Fix an overloaded server](/overloaded-server/) for more tips.
+{% endAside %}
+
+### Route users to a nearby CDN
+
+A Content Delivery Network (CDN) is a network of servers distributed in many different locations. If
+the content on your web page is being hosted on a single server, your website will load slower for
+users that are geographically further away because their browser requests literally have to travel
```
[the remark this answers is not in the document]
Comment (housseindjirdeh) [reply src/site/content/en/blog/optimize-lcp/index.md]: TIL!

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 8]
```diff
@@ -0,0 +1,644 @@
...
++   Deliver different assets based on network connection (adaptive serving)
++   Cache assets using a service worker
+
+### Optimize and compress images
+
+For many sites, images are the largest element in view when the page has finished loading. Hero
+images, large carousels or banner images are all common examples of this.
+
+<figure class="w-figure">
+  <img width="300" src="./image-largest-element.png">
```
[the remark this answers is not in the document]
Comment (housseindjirdeh) [reply src/site/content/en/blog/optimize-lcp/index.md:424]: Updated, design.google is perfect!

File: src/site/content/en/blog/optimize-lcp/index.md (hunk 2 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/optimize-lcp/index.md:645]: Updated to design.google here as well :)

**Comment by housseindjirdeh** [comment]
Thanks folks, updated and used all the suggestions.

@kaycebasques These will be linked to from within /metrics, but they're going to live in /blog. They're mostly repeating many of the existing guidance and aren't going to be used as the main canonical posts.

@philipwalton Updated the graphic to use "Needs Improvement" 👍🏾

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 9]
```diff
@@ -0,0 +1,653 @@
...
+(usually more meaningful) content on the page to render. A large hero image that takes up most of
+the device screen is a good example, and could load much later than early DOM content that renders
+on the page such as text.
+
+[Largest Contentful Paint](/lcp) (LCP) is a newer [Core Web
+Vitals](/metrics) metric and measures when the largest content element in the
+viewport becomes visible. It can be used to determine when the main content of the page has finished
+rendering on the screen.
+
+<img class="w-screenshot w-screenshot--filled" src="lcp-distributions.png" alt="Good LCP values are
```
Comment (philipwalton) [inline src/site/content/en/blog/optimize-lcp/index.md:40]: Do you think it makes sense to use the same markup and image I have here? This should be fine as long as my PR is merged first: https://github.com/GoogleChrome/web.dev/pull/2764/files#diff-cef065d40e06c392ef447cfba4a0855aR70-R76

cc: @addyosmani

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 10]
```diff
@@ -0,0 +1,653 @@
...
+
+One factor contributing to a poor user experience is how long it takes a user to see any content
+rendered to the screen. [First Contentful Paint](/fcp) (FCP) measures how long it
+takes for initial DOM content to render, but it does not capture how long it took the largest
+(usually more meaningful) content on the page to render. A large hero image that takes up most of
+the device screen is a good example, and could load much later than early DOM content that renders
+on the page such as text.
+
+[Largest Contentful Paint](/lcp) (LCP) is a newer [Core Web
+Vitals](/metrics) metric and measures when the largest content element in the
```
Comment (philipwalton) [inline src/site/content/en/blog/optimize-lcp/index.md:36]: Can you remove "newer"? Just say it's a Core Web Vital.

Also, can you link to `/vitals/`, which should be live before you merge this PR?

File: src/site/content/en/blog/optimize-lcp/index.md (hunk 9 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/optimize-lcp/index.md]: Done. Will check and merge after your PR gets merged

**Comment by housseindjirdeh** [comment]
@kaycebasques Updated a few more tiny things per some more review comments. I also [removed](https://github.com/GoogleChrome/web.dev/pull/2701/commits/e956161b4c301719c8c0d5d7a5b92db9f8683291#diff-2f6f9f076d3dba32b321a55cdf157579L279) a few sections that I plan on moving to the _Optimize FID_ post. 

Will merge this next week as soon (or right before) the launch 👍
